notifications_utils/logging.py

```diff
@@ -40,43 +43,49 @@ def init_app(app, statsd_client=None, extra_filters: Sequence[logging.Filter] =
     app.config.setdefault("NOTIFY_APP_NAME", "none")
     app.config.setdefault("NOTIFY_LOG_PATH", "./log/application.log")
     app.config.setdefault("NOTIFY_RUNTIME_PLATFORM", None)
+    app.config.setdefault("NOTIFY_LOG_IGNORE_PATH_LIST", "/_status;/metrics")
     app.config.setdefault(
         "NOTIFY_REQUEST_LOG_LEVEL",
         "CRITICAL" if app.config["NOTIFY_RUNTIME_PLATFORM"] == "paas" else "NOTSET",
     )
 
+    log_ignore_path_dict = {path: True for path in app.config["NOTIFY_LOG_IGNORE_PATH_LIST"].split(";")}
+
     @app.before_request
     def before_request():
         # annotating this onto request instead of flask.g as it probably shouldn't
         # be inheritable from a request-less application context
         request.before_request_real_time = time.perf_counter()
 
-        # emit an early log message to record that the request was received by the app
-        context = _common_request_extra_log_context()
-        current_app.logger.getChild("request").log(
-            logging.DEBUG,
-            "Received request %(method)s %(url)s",
-            context,
-            extra=context,
-        )
+        if request.path not in log_ignore_path_dict:
+            # emit an early log message to record that the request was received by the app
+            context = _common_request_extra_log_context()
+            current_app.logger.getChild("request").log(
+                logging.DEBUG,
+                "Received request %(method)s %(url)s",
+                context,
+                extra=context,
+            )
```

Comment:
I would leave these because they're at `DEBUG` level anyway.

Reply:
I've left the messages as DEBUG and I've left the post request message as DEBUG.